randovania/games/fusion/logic_database/Sector 5 ARC.txt

```diff
@@ -896,21 +896,29 @@ Extra - room_id: [28]
   * Extra - door_idx: (63,)
   > Door to Zebesian Waters
       Any of the following:
-          Missiles ≥ 10 or Combat (Intermediate) or Can Use Power Bombs
+          Combat (Intermediate) or Can Use Power Bombs
           Damage Boosts (Intermediate) and Normal Damage ≥ 250
-          Charge Beam and Plasma Beam
+          All of the following:
+              Charge Beam and Plasma Beam
+              Wave Beam or Wide Beam
```

Comment:
This feels like overengineering and trying to be a little too safe. IMO we should just `Charge Beam and Have Any Beam Upgrade`.